hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -73,6 +76,7 @@
     private final ILogger logger;
     private final ScheduledExecutorService scheduler;
     private final Address thisAddress;
+    private final ConcurrentMap<Address, AtomicLong> lastHeartbeatPerMember = new ConcurrentHashMap<Address, AtomicLong>();
```

Comment:
does it have to be concurrent map? it's always run on the same thread, isn't it?